.goreleaser.yml

```diff
@@ -27,32 +27,56 @@ builds:
       - amd64
     binary: '{{ .ProjectName }}_v{{ .Version }}_{{ .Env.API_VERSION }}_{{ .Os }}_{{ .Arch }}'
   -
+    id: linux-builds
     mod_timestamp: '{{ .CommitTimestamp }}'
     flags:
       - -trimpath #removes all file system paths from the compiled executable
     ldflags:
       - '-s -w -X {{ .ModulePath }}/version.Version={{.Version}} -X {{ .ModulePath }}/version.VersionPrerelease= '
     goos:
-      - netbsd
-      - solaris
-      - openbsd
-      - freebsd
-      - windows
       - linux
-      - darwin
     goarch:
       - amd64
       - '386'
       - arm
       - arm64
     ignore:
-      - goos: openbsd
-        goarch: arm64
-      - goos: darwin
-        goarch: '386'
       - goos: linux
         goarch: amd64
     binary: '{{ .ProjectName }}_v{{ .Version }}_{{ .Env.API_VERSION }}_{{ .Os }}_{{ .Arch }}'
+  -
+    id: darwin-builds
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath #removes all file system paths from the compiled executable
+    ldflags:
+      - '-s -w -X {{ .ModulePath }}/version.Version={{.Version}} -X {{ .ModulePath }}/version.VersionPrerelease= '
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    binary: '{{ .ProjectName }}_v{{ .Version }}_{{ .Env.API_VERSION }}_{{ .Os }}_{{ .Arch }}'
+  -
+    id: other-builds
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath #removes all file system paths from the compiled executable
+    ldflags:
+      - '-s -w -X {{ .ModulePath }}/version.Version={{.Version}} -X {{ .ModulePath }}/version.VersionPrerelease= '
+    goos:
+      - netbsd
+      - openbsd
+      - freebsd
+      - windows
+    goarch:
+      - amd64
+      - '386'
+      - arm
+    ignore:
+      - goos: windows
```

Comment:
This looks like it'll exclude ARM Windows now which it wasn't before, was this intentional?

Reply:
If we're on the subject of consistency, Windows ARM is not a release target for Packer as well, so it's actually a good move if we were delivering the plugin on this combination, as Packer is not supported there.

Reply:
Yes, this is intentional. Windows ARM doesn't appear in the release matrix for Packer core or any of the other HashiStack products. I believe it was part of the plugins as a side effect of switching to goreleaser.

Reply:
Note: looking at other plugins (i.e. Amazon for example), I see it releases on Windows ARM/ARM64. I guess your changes in this PR can be replicated for the others if we want to have the same compatibility matrix as Packer core.

Reply:
> Note: looking at other plugins (i.e. Amazon for example), I see it releases on Windows ARM/ARM64. I guess your changes in this PR can be replicated for the others if we want to have the same compatibility matrix as Packer core.

Yeah, this is something we should probably work on next.